docs/en/Running-Kylin-on-Alluxio.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: global
+title: Running Apache Kylin on Alluxio
+nickname: Apache Kylin
+group: Frameworks
+priority: 2
+---
+
+This guide describes how to run [Apache Kylin](http://kylin.apache.org/) on Alluxio, so
+that you can speed up the cube build phase in Kylin.
```

Comment:
I am not familar with Kylin, Does Alluxio only affect the build phase, or does it also help queries?